ci-operator/step-registry/code-ready/snc/e2e/test/code-ready-snc-e2e-test-commands.sh

```diff
@@ -38,6 +38,8 @@ function run-tests() {
   # wait till the cluster is stable
   sleep 5m
   export KUBECONFIG=crc-tmp-install-data/auth/kubeconfig
+  # Remove all the failed Pods
+  oc delete pods --field-selector=status.phase=Failed -A
```

Comment:
I'm not familiar enough with the installer / node affinity to understand what this is about :(

Reply:
@cfergeau https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#node-affinity have details about node affinity, problem is it is not consistent like the pods which are part of ingress go to this state sometime and sometime it doesn't and when it does it will make the script to run forever.

Reply:
This emphasizes the fact that, at the end of snc.sh script, the cluster is not clean. We should fix that but it's more a long term action.